source-postgres/docker-entrypoint-initdb.d/init-user-db.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 set -e
 
-# The alter system command at the end is redundant due command line settings. 
-#  Included as an example of how to set this without using the command line. 
+# The alter system command at the end is redundant due command line settings.
+#  Included as an example of how to set this without using the command line.
 psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
     CREATE USER flow_capture WITH PASSWORD 'flow_capture' REPLICATION;
+    ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES to flow_capture;
```

Comment:
This is a good addition, and should probably be added to the setup instructions in `README.md` and our docs as well.

Reply:
Ah, I just noticed you mentioned that in the Slack thread too.